### PR TITLE
VS19 Win updates for First Web API tutorial

### PR DESCRIPTION
**Pull request by wadepickett**
Updates for screenshots and some doc instruction for Win VS19 only.  Once the windows version publishes successfully I will work on VSCode and MacOS for this.  

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-2.2&branch=pr-en-us-12989&tabs=visual-studio)

Fixes # 12675

Also tested project sample for VS19 win.  Project code does not require any change.

### CODE REVIEW

**Comment by wadepickett** [comment]
@tdykstra, could you take a look to see if I did this right?  Many thanks!

**Comment by wadepickett** [comment]
Acrolinx score 96

**Comment by Rick-Anderson** [comment]
> Acrolinx score 96

Some Acrolinx suggestions are bad. Most larger docs should come in 80 - 85% -

File: aspnetcore/tutorials/first-web-api.md [hunk 1]
```diff
@@ -53,15 +53,16 @@ The following diagram shows the design of the app.
 # [Visual Studio](#tab/visual-studio)
 
 * From the **File** menu, select **New** > **Project**.
-* Select the **ASP.NET Core Web Application** template. Name the project *TodoApi* and click **OK**.
-* In the **New ASP.NET Core Web Application - TodoApi** dialog, choose the ASP.NET Core version. Select the **API** template and click **OK**. Do **not** select **Enable Docker Support**.
+* Select the **ASP.NET Core Web Application** template and click **Next**.
+* Name the project *TodoApi* and click **Create**.
+* In the **Create a new ASP.NET Core Web Application** dialog, choose the ASP.NET Core version. Select the **API** template and click **Create**. **Don't** select **Enable Docker Support**.
```
[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/tutorials/first-web-api.md]: And the version should be 2.2

[the remark this answers is not in the document]
Comment (wadepickett) [reply aspnetcore/tutorials/first-web-api.md]: Good point. I will update per the suggestion.  Thanks!

**Comment by wadepickett** [comment]
Thanks for the comments!  Appreciated and helps me judge where to best spend my time.
Postman itself recently updated and its UI changed around.  At first I wasn't going to update the postman screenshots figuring it was already clarified in the doc that the URL might be different.  However ,with both the URL and the Postman UI changing it was overall offputting enought that it would require the audiance visually sort through it too long.

For Acrolinx, I thought I would just make a habit of stating whatever I found the score as a way to check off that I took that step.  The actual number was neither here nor there for me since yes,  there are certainly a whole lot of false positives.

**Comment by wadepickett** [comment]
@tdykstra, patch in instructing to "confirm that **.NET Core** and **ASP.NET Core 2.2** are selected".  Thanks.   Good to pr/merge?

**Pull request merged**